helm/korifi/values.yaml

```diff
@@ -68,6 +68,29 @@ api:
     host: ""
     caCert: ""
 
+route-controller:
+  image: cloudfoundry/korifi-route-controller:latest
+
+  replicas: 1
+  nodeSelector: {}
+  tolerations: []
+  resources:
+    limits:
+      cpu: 1000m
+      memory: 1Gi
+    requests:
+      cpu: 50m
+      memory: 100Mi
+  processDefaults:
+    memoryMB: 1024
+    diskQuotaMB: 1024
+  taskTTL: 30d
+
+  namespaceLabels: {}
+  extraVCAPApplicationValues: { }
+  maxRetainedPackagesPerApp: 5
+  maxRetainedBuildsPerApp: 5
```

Comment:
Those are probably copied from the `controllers` config, but I don't think they would be needed in a routing controller. If so, the user should probably not have to maintain them twice. But it sounds as if there are not routing relevant.